bin/compile

```diff
@@ -21,23 +21,32 @@ BUILD_DIR=$1
 CACHE_DIR=$2
 ENV_DIR=$3
 BUILDPACK_DIR="$(dirname "$(dirname "$0")")"
+TAILSCALE_INSTALL="${TAILSCALE_INSTALL:-"true"}"
+USE_RAILS_BIN_WRAPPERS="${USE_RAILS_BIN_WRAPPERS:-"true"}"
 
-TAILSCALE_VERSION=1.74.0
+# https://github.com/tailscale/tailscale/releases
+# https://pkgs.tailscale.com/stable/#static
+TAILSCALE_VERSION="${TAILSCALE_VERSION:-1.80.2}"
 TAILSCALE_BUILD_ARCH=amd64
 TAILSCALE_SOURCE_URL="https://pkgs.tailscale.com/stable/tailscale_${TAILSCALE_VERSION}_${TAILSCALE_BUILD_ARCH}.tgz"
 TAILSCALE_INSTALL_DIR="$BUILD_DIR/vendor/tailscale"
 
-PROXYCHAINS_VERSION=4.17
+# https://github.com/rofl0r/proxychains-ng/releases
+PROXYCHAINS_VERSION="${PROXYCHAINS_VERSION:-4.17}"
 PROXYCHAINS_SOURCE_URL="https://github.com/rofl0r/proxychains-ng/archive/refs/tags/v$PROXYCHAINS_VERSION.tar.gz"
 PROXYCHAINS_INSTALL_DIR="$BUILD_DIR/vendor/proxychains-ng"
 
-log "Installing Tailscale"
-mkdir -p $TAILSCALE_INSTALL_DIR
-export PATH="$TAILSCALE_INSTALL_DIR:$PATH"
-curl -sL $TAILSCALE_SOURCE_URL \
-  | tar -zxf - -C "$TAILSCALE_INSTALL_DIR" --strip=1 tailscale_${TAILSCALE_VERSION}_${TAILSCALE_BUILD_ARCH}/tailscaled tailscale_${TAILSCALE_VERSION}_${TAILSCALE_BUILD_ARCH}/tailscale
-chmod +x "$TAILSCALE_INSTALL_DIR/tailscale"
-chmod +x "$TAILSCALE_INSTALL_DIR/tailscaled"
+if [ "$TAILSCALE_INSTALL" != "true" ]; then
```

Comment:
```
heroku config:set TAILSCALE_INSTALL=false TAILSCALE_VERSION=1.80.2 USE_RAILS_BIN_WRAPPERS=false PROXYCHAINS_VERSION=4.17 -a xbe-clone

```

Reply:
https://pkgs.tailscale.com/stable/#static vs. https://pkgs.tailscale.com/stable/#ubuntu-jammy

https://github.com/heroku/heroku-buildpack-apt

https://github.com/heroku/heroku-buildpack-apt/pull/41

Reply:
Looking at https://tailscale.com/blog/sisyphean-dns-client-linux and https://github.com/tailscale/tailscale/issues/8829

Reply:
also see https://github.com/tailscale/github-action/blob/2b498977f3e4dae1a5c7da1f138a9e6163220d65/action.yml

Reply:
and see https://tailscale.com/kb/1112/userspace-networking

Reply:
noting https://github.com/brettpetch/hosted-scripts/tree/363a95ebfa4ccf76f15f8175fcc15cdefd58dba7 https://github.com/tailscale/codespace/blob/44307894cef3057589a9b79f25ea1770682a050e/src/tailscale/install.sh#L1-L71  https://github.com/green-got/tailscale-github-action/blob/0566793c2b1cf6ce27805a7ede7a53714333f028/action.yml https://github.com/tailscale/github-action/blob/5d536117a7378beffcd6d8523ce73c50fa98684d/action.yml https://github.com/search?q=+content%3Ahello.ts.net+path%3Atest&type=code